case_studies/glenaeon_rudolf_steiner_school/03_board_meeting/recap.md

```diff
@@ -13,7 +13,7 @@ One of the things I was asked to do was write a short proposal for our club.
 > We would like to start a student run code club within the school. It would meet
 > every Wednesday afternoon from 3:30-4:30 in the library. Ideally we would get
 > Chris to supervise the meetings. We would roughly follow the hackEDU
```

Comment:
I'd leave them as they are for historical reference. However, add a note to
the `case_studies/README.md` file.

On Tue, Nov 24, 2015 at 11:31 AM Selynna Sun <notifications@github.com>
wrote:

> In case_studies/glenaeon_rudolf_steiner_school/03_board_meeting/recap.md
> <https://github.com/hackclub/hackclub/pull/511#discussion_r45682739>:
>
> > @@ -13,7 +13,7 @@ One of the things I was asked to do was write a short proposal for our club.
> >  > We would like to start a student run code club within the school. It would meet
> >  > every Wednesday afternoon from 3:30-4:30 in the library. Ideally we would get
> >  > Chris to supervise the meetings. We would roughly follow the hackEDU
>
> My bad I didn't realize they were other clubs' notes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/511/files#r45682739>.
>